Assignment2/amcarrero-AdrianMora.rdf

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+    <rdf:Description rdf:about="#Class01">
+	    <includes rdf:resource="#Sensor029"/>
+	    <includes rdf:resource="#Computer101"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+	    <hasMeasurement rdf:resource="#Measurement8401"/>
+    </rdf:Description>
+	<rdf:Description rdf:about="#Measurement8401">
+	    <hasTemperature> 29 </hasTemperature>
+	    <atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2010-06-12T12:00:12</atTime>
```

Comment:
This should be dateTime

Reply:
Just committed some changes, I hope it solves the problems.